packages/fxa-auth-server/lib/senders/emails/templates/inactiveAccountSecondWarning/includes.json

```diff
@@ -0,0 +1,10 @@
+{
+  "subject": {
+    "id": "inactiveAccountSecondWarning-subject",
+    "message": "Action required: Account deletion in 7 days"
+  },
+  "action": {
+    "id": "inactiveAccountSecondWarning-action",
+    "message": "Sign in to keep your account"
+  }
```

Comment:
The `preheader` content ("Sign in to keep your account") can be added here as `"preview"`, similar to https://github.com/mozilla/fxa/blob/7e5238905ad65a6fcdca227d2956c49d1df8cb6e/packages/fxa-auth-server/lib/senders/emails/templates/verifyShortCode/includes.json#L6-L9

I realize I forgot to add this to the PR for the first email - would you mind adding to both? 🙏